x-pack/filebeat/processors/decode_cef/keys.ecs.go

```diff
@@ -118,3 +130,8 @@ var ecsExtensionMapping = map[string]mappedField{
 	},
 	"type": {Target: "event.kind"},
 }
+
+func ecsMAC(in *cef.Field) (interface{}, error) {
+	in.String = strings.ToUpper(strings.ReplaceAll(in.String, ":", "-"))
+	return in.String, nil
```

Comment:
I doubt it matters, but none of the other `Translate` funcs mutate `*cef.Field`. They only return the modified value. Perhaps it should be a straight return without the set.

Reply:
Fixed and added docs to `mappedField`.